### ISSUE TITLE
Conv2d_transpose requires asymmetric padding which the CUDA EP currently does not support

### ISSUE THREAD

**Issue by ultimatedigiman** [onnx/tensorflow-onnx#1915]
**Describe the bug**
After successfully converting a tensorflow frozen graph(.pb) into a .oonx model, I encountered a warning while inferring the onnx model with onnxruntime-gpu in python.
The warning says the **conv2d_transpose** op is not supported because it requires **asymmetric padding** which the CUDA EP currently does not support.

But it's strange that I found  **ConvTranspose** is in [Supported ONNX Operators](https://github.com/onnx/onnx-tensorrt/blob/main/docs/operators.md) and it seems [the asymmetric padding issue](https://github.com/microsoft/onnxruntime/pull/4627) is already solved.

**System information**
- OS Platform and Distribution: Linux Ubuntu 20.04
- Tensorflow Version: 1.15
- Python version: 3.6
- opset: 13
- onnxruntime-gpu version: 1.10.0

**Screenshots**

![transpose warning](https://user-images.githubusercontent.com/67790229/163530444-9dbe1dfa-ac7c-49b4-bdf4-7bf237103327.png)

**Additional context**
I found `tf.nn.conv2d_transpose` has been tested in this repo with padding="VALID",while I use padding="SAME".
https://github.com/onnx/tensorflow-onnx/blob/f64772ce166ea2a0402524d9741b2fb71e5663df/tests/test_backend.py#L579-L590

**Comment by fatcat-z** [comment]
The warning happened during inferring the onnx model by onnxruntime-gpu, so probably you should go to https://github.com/microsoft/onnxruntime to raise this issue.

**Comment by ultimatedigiman** [comment]
[The onnxruntime repo](https://github.com/microsoft/onnxruntime/issues/11312) says **asymmetric padding** in `Conv` is already supported, but **asymmetric padding** in `ConvTranspose` is not yet supported.

**Comment by fatcat-z** [comment]
This is an ONNXRuntime issue so close it.
